dnsapi/dns_cx.sh

```diff
@@ -82,7 +82,7 @@ existing_records() {
     return 1
   fi
   count=0
-  seg=$(printf "%s\n" "$response" | _egrep_o "{[^\{]*host\":\"$_sub_domain\"[^\}]*\}")
+  seg=$(printf "%s\n" "$response" | _egrep_o "{[^{]*host\":\"$_sub_domain\"[^}]*}")
```

Comment:
see my fix here:  https://github.com/Neilpang/acme.sh/commit/5be3f22d06cd62397395b0de4e76f3f1d3c27405

It's not working on solaris now, I'm still fixing.